plugins/lookup/consul_kv.py

```diff
@@ -71,6 +70,13 @@
         ini:
           - section: lookup_consul
             key: client_cert
+      url:
+        description: "The target to connect to, should look like this: C(https://my.consul.server:8500)."
```

Comment:
```suggestion
        description: "The target to connect to, should look like this: C(https://my.consul.server:8500)."
        type: str
        version_added: 1.0.0
```
I assume `type: str` is correct?

Reply:
Yes, it's correct